core/server/fluxruntime.go

```diff
@@ -49,7 +58,8 @@ func (cs *coreServer) ListFluxRuntimeObjects(ctx context.Context, msg *pb.ListFl
 		list := &appsv1.DeploymentList{}
 
 		if err := clustersClient.List(ctx, clusterName, list, opts, client.InNamespace(fluxNs.Name)); err != nil {
-			return nil, fmt.Errorf("could not list deployments in namespace %s: %w", fluxNs.Name, err)
+			respErrors = append(respErrors, &pb.ListError{ClusterName: clusterName, Namespace: fluxNs.Name, Message: fmt.Sprintf("%s, %s", ErrListingDeployments.Error(), err)})
```

Comment:
@luizbafilho I tried to help in adding partial response to ListFluxRuntimeObjects but I couldn't find an easy way to make this line fail, would you have any suggestion?

Reply:
ok ok